src/cata_utility.h

```diff
@@ -40,7 +40,9 @@ double logarithmic( double t );
 double logarithmic_range( int min, int max, int pos );
 
 int bound_mod_to_vals( int val, int mod, int max, int min );
+std::string velocity_units( bool to_alternative_units = false );
```

Comment:
Nobody knows what "alternative_units" is. Nobody knows what the difference between `velocity_units(true)` and `velocity_units(false)` is.

I suspect having separate functions (`velocity_units_vehicle`, maybe put it into vehicle class as static function named `vehicle::velocity_units` and `wind_velocity_units`) would be better.